packages/evo-marko/src/tags/evo-skeleton/test/__snapshots__/test.server.ts.snap

```diff
@@ -0,0 +1,80 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
I see the snapshots, but I don't see the tests
Lets copy them over just so we don't forget them. (and rename them to typescript files)